clearwater-cassandra/usr/share/clearwater/infrastructure/scripts/cassandra_schemas/run_cassandra_schemas

```diff
@@ -12,6 +12,9 @@ for SCHEMA in $(ls -1 /usr/share/clearwater/cassandra-schemas/* 2>/dev/null)
 do
   if [ -f "$SCHEMA" ]; then
     $SCHEMA
+    rc=$?
+    if [ $rc != 0 ]; then
```

Comment:
Do we have to look at the rc?
We're going to return it either way (since if the rc is 0, the script will complete successfully, so a rc of 0 is returned, right?), so couldn't we just return it without looking at it?